npm/webpack-dev-server/src/loader.ts

```diff
@@ -18,7 +18,7 @@ const makeImport = (file: Cypress.Cypress['spec'], filename: string, chunkName:
   const magicComments = chunkName ? `/* webpackChunkName: "${chunkName}" */` : ''
 
   return `"${filename}": {
-    shouldLoad: () => document.location.pathname.includes("${encodeURI(file.absolute)}"),
```

Comment:
I guess we don't need this. If it passes CI, I'm pretty confident we are good to go.

Reply:
We already had spaces: https://github.com/cypress-io/cypress/pull/20575/files#diff-f5c1d2cf3b3dd9c9d3c0464a216d71fdf03609a366cfe48406b699823ddce7f7R138

- `...`: https://github.com/cypress-io/cypress/pull/20575/files#diff-f5c1d2cf3b3dd9c9d3c0464a216d71fdf03609a366cfe48406b699823ddce7f7R119
- unicode: https://github.com/cypress-io/cypress/pull/20575/files#diff-f5c1d2cf3b3dd9c9d3c0464a216d71fdf03609a366cfe48406b699823ddce7f7R100